### PR TITLE
Remove PPro isLaunched FF

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1207185787167219/f

### Description
Remove PPro isLaunched FF now that PPro is fully rolled out.

### Steps to test this PR
- [x] smoke test app update from VPN beta user
- [x] smoke test fresh install
- [x] smoke test fresh install and subscribe to PPro

### CODE REVIEW

**Comment by aitorvs** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4479?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4479** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4479?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4478** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4478?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @aitorvs and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementActivity.kt [hunk 1]
```diff
@@ -152,14 +147,10 @@ class NetworkProtectionManagementActivity : DuckDuckGoActivity() {
 
         binding.about.aboutFaq.setClickListener {
             lifecycleScope.launch(dispatcherProvider.io()) {
-                if (subscriptions.isEnabled()) {
-                    globalActivityStarter.start(
-                        this@NetworkProtectionManagementActivity,
-                        WebViewActivityWithParams(url = VPN_HELP_CENTER_URL, screenTitle = getString(R.string.netpFaqTitle)),
-                    )
-                } else {
-                    globalActivityStarter.start(this@NetworkProtectionManagementActivity, NetPFaqsScreenNoParams)
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementActivity.kt:161]: NIT: NetPFaqsScreenNoParams and NetpFaqActivity can now be removed as well

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetworkProtectionAccessStateImpl.kt [hunk 2]
```diff
@@ -39,11 +37,10 @@ class NetworkProtectionAccessStateImpl @Inject constructor(
     private val networkProtectionState: NetworkProtectionState,
     private val dispatcherProvider: DispatcherProvider,
     private val netpSubscriptionManager: NetpSubscriptionManager,
-    private val subscriptions: Subscriptions,
 ) : NetworkProtectionAccessState {
 
     override suspend fun getState(): NetPAccessState = withContext(dispatcherProvider.io()) {
-        if (isTreated()) {
+        if (true) {
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetworkProtectionAccessStateImpl.kt:43]: Why keep this?

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementActivity.kt (hunk 1 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementActivity.kt:161]: done

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetworkProtectionAccessStateImpl.kt (hunk 2 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetworkProtectionAccessStateImpl.kt]: argh, I refactored that step by step and forgot to let the IDE simplify that. Done

**Review by karlenDimla — APPROVED** [review]
Some nitpicks

**Comment by aitorvs** [comment]
### Merge activity

* **Apr 29, 8:46 AM EDT**: @aitorvs started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/4479).

**Pull request merged**